types/i18next-express-middleware/index.d.ts

```diff
@@ -5,104 +5,130 @@
-         */
-        constructor(services?: any, options?: Object, allOptions?: Object);
+    addDetector(detector: i18nextCustomDetection): void
 
-        /**
-         * @summary Adds detector.
-         * @param {i18nextCustomDetection} detector The detector to add.
-         */
-        addDetector(detector: i18nextCustomDetection): void;
+    // NOTE: add documentation
+    cacheUserLanguage(
+      req: express.Request,
+      res: express.Response,
+      detectionOrder: any
+    ): void
 
-        // NOTE: add documentation
-        cacheUserLanguage(req: express.Request, res: express.Response, detectionOrder: any): void;
-
-        /**
-         * @summary Detects the language.
-         * @param {Request}         req             The HTTP request.
-         * @param {Response}        res             The HTTP response.
-         * @param {detectionOrder}  detectionOrder  The detection order.
-         */
-        detect(req: express.Request, res: express.Response, detectionOrder: any): void;
-
-        /**
-         * @summary Initializes class.
-         * @param {any}     services    The services.
-         * @param {Object}  options     The options.
-         * @param {Object}  allOptions  The all options.
-         */
-        init(services: any, options?: Object, allOptions?: Object): void;
-    }
-
-    export function getResourcesHandler(i18next: i18next.i18n, options: Object): express.Handler;
-    export function handle(i18next: i18next.i18n, options?: Object): express.Handler;
+    /**
+     * @summary Detects the language.
+     * @param {Request}         req             The HTTP request.
+     * @param {Response}        res             The HTTP response.
+     * @param {detectionOrder}  detectionOrder  The detection order.
+     */
+    detect(
+      req: express.Request,
+      res: express.Response,
+      detectionOrder: any
+    ): void
 
     /**
-     * @summary Gets handler for missing key.
-     * @param {I18nextStatic}   i18next The i18next.
-     * @param {Object}          options The options.
-     * @return {express.Handler} The express handler.
+     * @summary Initializes class.
+     * @param {any}     services    The services.
+     * @param {Object}  options     The options.
+     * @param {Object}  allOptions  The all options.
      */
-    export function missingKeyHandler(i18next: i18next.i18n, options: Object): express.Handler;
+    init(services: any, options?: Object, allOptions?: Object): void
+  }
+
+  export function getResourcesHandler(
+    i18next: i18next.i18n,
+    options: Object
+  ): express.Handler
+  export function handle(
+    i18next: i18next.i18n,
+    options?: Object
+  ): express.Handler
+
+  /**
+   * @summary Gets handler for missing key.
+   * @param {I18nextStatic}   i18next The i18next.
+   * @param {Object}          options The options.
+   * @return {express.Handler} The express handler.
+   */
+  export function missingKeyHandler(
```

Comment:
@cyrilschumacher besides updating the definition with prettier, this is the only line I changed.